src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -3178,6 +3178,12 @@
   <data name="ShowCheckboxesWhenSelectingItems" xml:space="preserve">
     <value>Show checkboxes when selecting items</value>
   </data>
+  <data name="EditPathDescription" xml:space="preserve">
+    <value>Go to path bar</value>
```

Comment:
Can you update this to match the specs sheet?

Reply:
done